data/translations/en/state-data.json

```diff
@@ -1,471 +1,925 @@
 {
   "al": {
     "hp_link": "https://www.sos.alabama.gov/alabama-votes",
+    "hp_link_english_only": "false",
+    "hp_link_spanish_selection": "false",
```

Comment:
This one also needs further investigation. I don't think we will be creating a new property for each language so we might need to figure out what this is doing and how to make it more scalable to additional languages.

Reply:
Ask Laura about this in relationship to the selecciones link property.

Reply:
We will be updating the spanish selection key to `hp_link_language_selection`